packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -84,9 +80,31 @@ export interface Props {
   footerOptions?: TableFooterCalc;
   footerValues?: FooterItem[];
   enablePagination?: boolean;
-  cellHeight?: TableCellHeight;
+  cellHeight?: schema.TableCellHeight;
   /** @alpha */
   subData?: DataFrame[];
   /** @alpha Used by SparklineCell when provided */
   timeRange?: TimeRange;
 }
+
+export interface CustomCellRendererProps {
```

Comment:
This interface may be still up to discussion not sure though if it makes sense to narrow this down (though maybe all you need is frame and index and other parts are just shortcuts) and nothing comes to my mind that should be added.

There may be some perf implications in case somebody wants to memoize the custom component. Then I would assume having just the `value` would mean we would not rerender rows/values that did not change, while with this it will rerender because of the `dataFrame` but you can mitigate it by custom `arePropsEqual` function so I think it's not an issue here.